org.mwc.debrief.help/cheat_sheets/GettingStarted/intro_composite.xml

```diff
@@ -10,6 +10,87 @@
 			make the bottom section of this pane taller by dragging the central
 			&apos;grab&apos; bar vertically upwards<i>NoPrint</i>
 		</intro>
+		<taskGroup kind="sequence" name="Welcome to Debrief NG" skip="true">
+			<intro>Welcome to Getting Started with Debrief. In this tutorial, the first in a series
```

Comment:
@alastrina123  - I'd rather we keep the tutorial content in the specific cheat sheet. Each tutorial is easier to edit if it's all together.  So, I'd rather these intro & onCompletion blocks just contain "summary" text that loosely describes what is coming, and what has been covered.